scripts/deploy.sh

```diff
@@ -30,10 +30,8 @@ env | grep -oP "^[^=]+" | sort
 
```

Comment:
Consider renaming to `release.sh` for consistency with the rest of the naming scheme. Would affect `deploy-manually.sh` and `deploy-webhook.sh` too